### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by Phil-Friderici**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,281 @@
...
+
+Using the default of 'NONE' will not add any rules.
+
+- *Default*: 'NONE'
+
+##### Example:
+```yaml
+tacacsplus::acl:
+  'acl_all_features':
+    permit: '127.0.0.1'
```
Comment (ghoneycutt) [inline README.md:34]: I don't think you could have duplicate key names in a hash

Comment (Phil-Friderici) [reply README.md:34]: https://github.com/ghoneycutt/puppet-module-tacacsplus/blob/prep_v1/spec/classes/init_spec.rb#L94-L123

spec test don't lie :)

Comment (ghoneycutt) [reply README.md:34]: the spec test shows an array of hashes. Could you please try a functional test with that hiera data to be sure?

Comment (Phil-Friderici) [reply README.md:34]: They show an hash ($acl) containing an array which contains hashes itself again. It's the array that allows multiple instances of the key value pairs. Took me some time too to realise this.

**Pull request merged**